app/questionnaire/questionnaire_schema.py

```diff
@@ -459,56 +472,54 @@ def get_individual_response_show_on_hub(self) -> bool:
         )
         return show_on_hub
 
-    def get_individual_response_individual_section_id(self) -> Optional[str]:
-        section_id: Optional[str] = self._questionnaire_json.get(
+    def get_individual_response_individual_section_id(self) -> str | None:
+        section_id: str | None = self._questionnaire_json.get(
             "individual_response", {}
         ).get("individual_section_id")
         return section_id
 
-    def get_title_for_section(self, section_id: str) -> Optional[str]:
+    def get_title_for_section(self, section_id: str) -> str | None:
         if section := self.get_section(section_id):
             return section.get("title")
 
-    def get_show_on_hub_for_section(self, section_id: str) -> Optional[bool]:
+    def get_show_on_hub_for_section(self, section_id: str) -> bool | None:
         if section := self.get_section(section_id):
             return section.get("show_on_hub", True)
 
-    def get_summary_for_section(self, section_id: str) -> Optional[ImmutableDict]:
+    def get_summary_for_section(self, section_id: str) -> ImmutableDict | None:
```

Comment:
Can we give returned `immutable dicts` top level arguments? There is few more below.